template/static/js/api.js

```diff
@@ -0,0 +1,96 @@
+/* util funcs */
```

Comment:
Use full words e.g., functions

Reply:
so i make a new pr to frontend for the nits?

Reply:
Nits can be fixed in this PR. No need to get my review again though.